src/pages/workspaces/workspace/ShareWorkspaceModal.js

```diff
@@ -108,72 +108,69 @@ export default ajaxCaller(class ShareWorkspaceModal extends Component {
     }, [
       h(IdContainer, [id => h(Fragment, [
         h(FormLabel, { htmlFor: id }, ['User email']),
-        h(AutocompleteTextInput, {
+        h(CreatableSelect, {
```

Comment:
What can we do to utilize a text input rather than a select? What a11y impacts does this have?
Would a combobox aria-label work for a select component?